modules/terrain/src/lib/parse-terrain.js

```diff
@@ -145,7 +145,7 @@ function getMartiniTileMesh(meshMaxError, width, terrain) {
  * @param {number} width width of the input data array
  * @param {number} height height of the input data array
  * @param {number[] | Float32Array} terrain elevation data
- * @returns {object} vertices and triangles data
+ * @returns {{vertices: Uint16Array, triangles: number[]}} vertices and triangles data
```

Comment:
I think `vertices` should be `number[]` too?
https://github.com/mapbox/delatin/blob/63289e2c5a2a8e7c26c36739cd1d2b9e21497f9c/index.js#L9